packages/swap/src/translators/constants.ts

```diff
@@ -0,0 +1,10 @@
+import {Swap} from '@yoroi/types'
+
+export const supported_protocols: ReadonlyArray<Swap.SupportedProtocol> = [
```

Comment:
for now, use camelCase to client, (I'm reconsidering that too)

Reply:
I think you meant, the client can see just the types?

Reply:
yes, but have renamed it for now